src/roaring.c

```diff
@@ -55,6 +55,39 @@ roaring_bitmap_t *roaring_bitmap_of(size_t n_args, ...) {
     return answer;
 }
 
+static inline int32_t minimum(uint32_t a, uint32_t b) { return (a < b) ? a : b; }
+
+static inline int32_t next_container_bound(uint32_t value) {
+    return (1<<16)*(value/(1<<16)) + (1<<16);
+}
+
+roaring_bitmap_t *roaring_bitmap_from_range(uint32_t min, uint32_t max, uint32_t step) {
+    if(step == 0)
+        return NULL;
+    roaring_bitmap_t *answer = roaring_bitmap_create();
+    if(max <= min)
```

Comment:
Yes, in Python `range(10, 3)` is valid code, which is simply an empty range.

I find this empty range nice, but it may hide integer overflows and thus produce some nasty bugs. So yes, returning `NULL` may be better. Do you have other concerns about this?